setup.cfg

```diff
@@ -1,7 +1,61 @@
 [metadata]
+name = ctapipe_io_nectarcam
+description = ctapipe plugin for reading NectarCam files
 long_description = file: README.md
-long_description_content_type = text/markdown
+long_description_content_type = text/markdown; charset=UTF-8
+author = Franca Cassol
+author_email = cassol@cppm.in2p3.fr
+license = MIT
+
+project_urls =
+    Bug Tracker = https://github.com/cta-observatory/ctapipe_io_nectarcam/issues
+    Source Code = https://github.com/cta-observatory/ctapipe_io_nectarcam
+
+classifiers =
+    Development Status :: 3 - Alpha
+    License :: OSI Approved :: MIT License
+    Intended Audience :: Science/Research
+    Topic :: Scientific/Engineering :: Astronomy
+    Topic :: Scientific/Engineering :: Physics
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.8
+
+[options]
+packages = find:
+package_dir =
+    = src
+python_requires = ~=3.8
+zip_safe = False
+install_requires =
+    astropy~=4.2
+    ctapipe~=0.12
+    numpy~=1.22
+    protozfits~=2.0
+    pytables>=3.7
```

Comment:
The package name on pypi is tables, without the py

Reply:
Conda calls it pytables though (so it is correct in environment.yml)

Reply:
Thanks a lot, @maxnoe ! How mean...